### PR TITLE
groups: lures for lure

### PR DESCRIPTION
**Pull request by arthyn**
This adds an entry into the invite people dialog screen so that people now see that they can invite others whether on urbit or off. This combines the editor in the previous PR with display so that the host can enable from either location. This also pulls the "invite people" link up into the group sidebar so it's more present. There are more locations in @urcades onboarding figma, but I thought we should just get this going first.

Also refactors the way we do dialogs so that there's less componentry to use out the gate and cleans up our scrollbars so they're transparent except for the thumb.

Screen Shot 2023-03-24 at 5 26 22 PM

**host**
Screen Shot 2023-03-24 at 5 26 13 PM

Screen Shot 2023-03-24 at 5 25 58 PM

**member**

Screen Shot 2023-03-24 at 5 26 29 PM
Screen Shot 2023-03-24 at 5 26 55 PM

### CODE REVIEW

**Review by zalberico — CHANGES_REQUESTED** [review]
I think the invite headers could be clearer, but I like the separate modal and where you have the link in the sidebar.

> Send Landscape Invite to Someone

I'd probably change this to: Send Invite Link to Friends Not on Urbit

> Invite people to group

I'd change this to: Send Invite to Friends Already on Urbit

Thoughts?

**Comment by urcades** [comment]
Here's a set of suggestions, and an interface to grant them some context:

---

NEW — **Invite someone from outside the Urbit network**
Courtesy of Tlon Corporation

Have friends, family or collaborators who aren’t on Urbit? You can now gift them an Urbit ID and onboard them to your group all in one free and easy sweep. Just copy and share the link below. 

---

**Invite by Urbit ID** 
(e.g. ~sampel-palnet) or display name. 
[Search]

---

Interface Example:

Invite to Group Modal

**Comment by arthyn** [comment]
updated to look like this: 
Screen Shot 2023-03-27 at 2 17 07 PM

File: ui/src/dms/MessagesSidebar.tsx [hunk 1]
```diff
@@ -140,7 +140,7 @@ export default function MessagesSidebar() {
   );
 
   return (
-    <nav className="flex h-full w-64 flex-none flex-col border-r-2 border-gray-50 bg-white">
```
Comment (patosullivan) [inline ui/src/dms/MessagesSidebar.tsx:143]: Why make this full?

**Review by patosullivan — APPROVED** [review]
Looks good! Just one style related comment, but I'm probably missing something.

**Review by jamesacklin — APPROVED** [review]
Degrades nicely if you don't have lure support installed (or the group host doesn't have it turned on).

**Review by zalberico — APPROVED** [review]
Thanks - Let's ship it!

File: ui/src/dms/MessagesSidebar.tsx (hunk 1 above)
Comment (arthyn) [reply ui/src/dms/MessagesSidebar.tsx:143]: bah I saw this comment and forgot lol

**Pull request merged**